modules/features/search/src/main/java/au/com/shiftyjelly/pocketcasts/search/SearchAutoCompleteResultsPage.kt

```diff
@@ -124,6 +127,22 @@ fun SearchAutoCompleteResultsPage(
                     }
                 }
             }
+
+            if (results.isNotEmpty()) {
+                item {
+                    TextP40(
+                        modifier = Modifier
+                            .clickable(
+                                onClick = {
+                                    onTermClick(SearchAutoCompleteItem.Term(searchTerm))
+                                },
+                            )
+                            .padding(vertical = 8.dp, horizontal = 16.dp),
+                        text = stringResource(LR.string.search_suggestions_view_all, searchTerm),
+                        color = MaterialTheme.theme.colors.primaryInteractive01,
+                    )
```

Comment:
```suggestion
                        color = MaterialTheme.theme.colors.primaryInteractive01,
                        maxLines = 1,
                    )
```
This is very minor but I wonder if we should limit it to 1 line.

| Before | After |
| --- | --- |
| <img width="1198" height="2531" alt="Image" src="https://github.com/user-attachments/assets/8c30db60-2e53-4dc6-8eea-1a9265fd6b47" /> | <img width="1198" height="2531" alt="Image" src="https://github.com/user-attachments/assets/8a8168f2-1347-4574-a1e5-4899ab000480" /> |

Reply:
great catch, updated!